README.md

```diff
@@ -37,6 +37,12 @@ Here's a command you can run from your terminal to generate a token via curl:
 
 Where USERNAME is your Github username. Save the token generated and paste it in the settings section under the token option.
 
+If OTP is enabled on your account, this will return 401 error code, use:
+
+    curl -v -u USERNAME -H "X-GitHub-OTP: OTPCODE" -X POST https://api.github.com/authorizations --data "{\"scopes\":[\"gist\"], \"note\": \"SublimeText 2/3 Gist plugin\"}"
+    
+Where OTPCODE is the code your quthenticator app shows you.
```

Comment:
Hey, good catch.
I don't have access to the branch anymore though, as I've deleted my fork (it's been a while since I created this PR).

So the only option if you want the change is to merge then fix.